nano/secure/common.cpp

```diff
@@ -304,6 +305,59 @@ nano::block_hash nano::pending_key::key () const
 	return account;
 }
 
+nano::unchecked_info::unchecked_info () :
+block (nullptr),
+account (0),
+modified (0),
+verified (nano::signature_verification::unknown)
+{
+}
+
+nano::unchecked_info::unchecked_info (std::shared_ptr<nano::block> block_a, nano::account const & account_a, uint64_t modified_a, nano::signature_verification verified_a) :
+block (block_a),
+account (account_a),
+modified (modified_a),
+verified (verified_a)
+{
+}
+
+void nano::unchecked_info::serialize (nano::stream & stream_a) const
+{
+	assert (block != nullptr);
+	nano::serialize_block (stream_a, *block);
+	nano::write (stream_a, account.bytes);
+	nano::write (stream_a, boost::endian::native_to_big (modified));
```

Comment:
There's not a lot of value in storing these as a specific endian-ness since we are going to delete this table on startup

Reply:
Seems valid, thanks! Removing endian stuff